py3status/modules/insync.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+"""
+Gets the current insync status
+
+Configuration parameters:
+    format: Display format to use *(default
+        '{status}'
+        )*
+
+Format status string parameters:
+    {status} Status of Insync
+
+@author Joshua Pratt <jp10010101010000@gmail.com>
+Thanks to @author Iain Tatch <iain.tatch@gmail.com> for the script that this is based on
```

Comment:
This should be either a) removed or b) reworded and moved into the main body of the docstring.